sections/06_discussion.md

```diff
@@ -631,7 +630,7 @@ These individual representations are further concatenated before or within
 fully-connected layers. FIDDLE [@tag:Eser2016_fiddle] is an example of a
 multimodal CNN that represents an ensemble of individual networks that take
 NET-seq, MNase-seq, ChIP-seq, RNA-seq, and raw DNA sequence as input to predict
-Transcription Start Site-seq. The combined model radically improves performance
+transcription start sites. The combined model radically improves performance
```

Comment:
We had `Transcription Start Site-seq` originally because that is emphasized in the FIDDLE paper.  For example, 

> we trained our ConvNet modules to predict TSS-seq data for individual input datasets

I believe they consistently use `TSS-seq` to emphasize the output is a continuous 500 bp distribution instead of a binary annotation.  However, I'm fine simplifying it and accepting your change unless someone else is strongly opposed.